sparse/tests/test_coo.py

```diff
@@ -1790,6 +1790,25 @@ def test_expand_dims(axis):
     np.testing.assert_equal(result.todense(), expected)
 
 
+@pytest.mark.parametrize(
+    "arr",
+    [
+        np.array([[0, 0, 1, 5, 3, 0], [1, 0, 4, 0, 3, 0], [0, 1, 0, 1, 1, 0]], dtype=np.int64),
+        np.array([[[2, 0], [0, 5]], [[1, 0], [4, 0]], [[0, 1], [0, -1]]], dtype=np.float64),
+    ],
+)
+@pytest.mark.parametrize("fill_value", [-1, 0, 1, 3])
+@pytest.mark.parametrize("axis", [0, 1, -1])
+@pytest.mark.parametrize("descending", [False, True])
+def test_sort(arr, fill_value, axis, descending):
```

Comment:
I'd add a `test_sort_random` which checks random values and edge cases such as completely dense, `nnz=0`, and differemt fill-values.

Reply:
Yes, right now I'm also working on `descending` for NumPy: https://github.com/numpy/numpy/pull/25437

Reply:
Sure! Added completely dense case to `test_sort` and fully empty as a separate test (with parametrization).